fs/littlefs/lfs_vfs.c

```diff
@@ -367,6 +368,15 @@ static int littlefs_open(FAR struct file *filep, FAR const char *relpath,
 
   relpath = littlefs_convert_path(relpath);
   oflags = littlefs_convert_oflags(oflags);
+  if (fs->readonly)
+    {
+      if (oflags != LFS_O_RDONLY)
```

Comment:
i suspect it isn't that simple.

for example, O_RDONLY|O_CREAT is expected to succeed if the file exists.

Reply:
but this should fail too

Reply:
> but this should fail too

?

can you explain a bit?